stories/core/SideNavigation.stories.tsx

```diff
@@ -82,3 +94,96 @@ export const ActiveItem: Story<SideNavigationProps> = (args) => {
     />
   );
 };
+
+export const Submenu: Story<SideNavigationProps> = (args) => {
+  const itemLabels = ['Home', 'Issues', 'Documents'];
+  const itemIcons = [
+    <SvgHome key={0} />,
+    <SvgFlag key={1} />,
+    <SvgFolderOpened key={2} />,
+  ];
+
+  const [activeItem, setActiveItem] = useState(2);
+  const [isSubmenuOpen, setIsSubmenuOpen] = useState(true);
+  const [activeSubItem, setActiveSubItem] = useState(0);
+
+  const mainItems = itemLabels.map((label, index) => (
+    <SidenavButton
+      key={index}
+      startIcon={itemIcons[index]}
+      isActive={activeItem === index}
+      onClick={() => {
+        if (label !== 'Documents') {
+          setActiveItem(index);
+          setActiveSubItem(-1);
+          setIsSubmenuOpen(false);
+        } else {
+          setIsSubmenuOpen((open) => !open);
+        }
+      }}
+    >
+      {label}
+    </SidenavButton>
+  ));
+
+  return (
+    <div style={{ display: 'flex', height: '100%' }}>
+      <SideNavigation
+        {...args}
+        expanderPlacement='bottom'
+        items={mainItems}
+        secondaryItems={[
+          <SidenavButton startIcon={<SvgSettings />} key={3}>
```

Comment:
It would be nice to have few items with sub-menus. You could add it to the Settings too.